config/Dockerfiles/fedoraci-runner/package-test/package-test.sh

```diff
@@ -175,6 +176,34 @@ if [[ `ls -1 tests*.yml 2>/dev/null | wc -l` == 0 ]]; then
     exit 1
 fi
 
+# Merge two results.yml files.
+#
+# This function merges previous results.yml into current results.yml.
+# If the two results.yml files are the same, it does nothing.
+#
+# Arguments:
+# - path to the previous results.yml file (e.g.: ${TEST_ARTIFACTS}/results.yml.prev)
+# - path to the current results.yml file
+function merge_results {
+
+    local old_results_path=${1}
+    local new_results_path=${2}
+
+    if ! [ -f "${old_results_path}" ] || ! [ -f "${new_results_path}" ]; then
```

Comment:
The ! should be inside the [ ], e.g.:
if [ ! -f "${old_results_path}" ] || [ ! -f "${new_results_path}" ]; then

Reply:
Good catch, fix incoming :+1:

Reply:
Fixed.